### PR TITLE
DAOS-4912 test: Enabled exclude/reintegrate into soak

### PR DESCRIPTION
**Pull request by mjean308**
The soak_harasser will enable the following:

harasser pairs:
exclude/reintegrate - dmg exclude/dmg reintegrate
svr_stop/srv_start - dmg drain/dmg system start
                     dmg start/dmg reintegrate

The test is writing/reading small and large files
with IOR and FIO and is using RP_2G1 objects.

The test can run up to 12 hours

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

**Comment by mjean308** [comment]
I had to abandon PR-3871 because the PR seemed to have been corrupted after a merge.

**Comment by shimizukko** [comment]
Sorry. I'm on vacation so not sure when I can do the review.

File: src/tests/ftest/util/dmg_utils.py [hunk 1]
```diff
@@ -844,17 +844,23 @@ def system_query(self, ranks=None, verbose=True):
         self.log.info("system_query data: %s", str(data))
         return data
 
-    def system_start(self):
+    def system_start(self, ranks=None):
         """Start the system.
 
+        Args:
+            force (bool, optional): whether to force the stop. Defaults to
+                False.
```
Comment (saurabhtandan) [inline src/tests/ftest/util/dmg_utils.py:852]: Minor: May be this got copied un-intentionally. Should be removed.

**Review by saurabhtandan — DISMISSED** [review]
LGTM

**Review by saurabhtandan — APPROVED** [review]
LGTM

File: src/tests/ftest/soak/soak_test_base.py [hunk 2]
```diff
@@ -413,19 +521,28 @@ def run_soak(self, test_param):
...
+                "<<Soak PASS %s: time until done %s>>", self.loop,
                 DDHHMMSS_format(self.end_time - time.time()))
-            # Create all specified pools
-            add_pools(self, pool_list)
+            # Create pool for jobs
+            add_pools(self, ["pool_jobs"])
             self.log.info(
                 "Current pools: %s",
                 " ".join([pool.uuid for pool in self.pool]))
+            # Initialize if harassers
```
Comment (shimizukko) [inline src/tests/ftest/soak/soak_test_base.py:531]: This comment doesn't really make sense. Typo?

**Review by shimizukko — APPROVED** [review]
LGTM

File: src/tests/ftest/soak/soak_test_base.py (hunk 2 above)
Comment (mjean308) [reply src/tests/ftest/soak/soak_test_base.py:531]: It should say Initialize if harassers enabled

File: src/tests/ftest/soak/soak_stress_2h.yaml [hunk 3]
```diff
@@ -7,13 +7,13 @@ hosts:
...
-    client_partition: daos_client
-    client_reservation:
+    client_partition: normal
+    client_reservation: daos-test
 orterun:
     allow_run_as_root: True
 # This timeout must be longer than the test_timeout param (+15minutes)
 # 2 hour test
-timeout: 2H15M
+timeout: 1H15M
```
Comment (johannlombardi) [inline src/tests/ftest/soak/soak_stress_2h.yaml:16]: not really 2h ...

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**